### ISSUE TITLE
[BUG] Commandline pihole -v on system w/o admin web interface

### ISSUE THREAD

**Issue by fmjensen** [pi-hole/pi-hole#1303]
**In raising this issue, I confirm the following (please check boxes, eg [X]) Failure to fill the template will close your issue:**

- [X] I have read and understood the [contributors guide](https://github.com/pi-hole/pi-hole/blob/master/CONTRIBUTING.md).
- [X] The issue I am reporting can be *replicated*
- [X] The issue I'm reporting isn't a duplicate (see [FAQs](https://github.com/pi-hole/pi-hole/wiki/FAQs), [closed issues](https://github.com/pi-hole/pi-hole/issues?utf8=%E2%9C%93&q=is%3Aissue%20is%3Aclosed%20), and [open issues](https://github.com/pi-hole/pi-hole/issues)).

**How familiar are you with the codebase?:**

_{5}_

---
**[BUG] Expected Behaviour:**
 ```pihole -v``` should show installed versions of Pi-Hole and Web-Admin

**[BUG | ISSUE] Actual Behaviour:**
ouput on RPi 3 installed without web interface as possible in the new version.:
```
me@rpiii:~ $ pihole -v
/opt/pihole/version.sh: linje 21: cd: /var/www/html/admin/: Ingen sådan fil eller filkatalog
::: Pi-hole version is v2.13.1 (Latest version is v2.13.1)
::: Web-Admin version is  (Latest version is v2.5.1)
me@rpiii:~ $ 
```
**[BUG] Steps to reproduce:**

use the CLI command ```pihole -v``` on system that was installed without the admin web interface.

**Comment by dschaper** [comment]
Thanks for that catch and report!

**Comment by fmjensen** [comment]
LGTM : Problem fixed at my RPi 3, now running v2.13.2
